extensions/resteasy-reactive/quarkus-resteasy-reactive-jaxb/deployment/src/main/java/io/quarkus/resteasy/reactive/jaxb/deployment/ResteasyReactiveJaxbProcessor.java

```diff
@@ -100,8 +111,16 @@ void registerClassesToBeBound(ResteasyReactiveResourceMethodEntriesBuildItem res
             boolean consumesXml = consumesXml(resourceInfo);
             boolean consumesMultipart = consumesMultipart(resourceInfo);
             if (consumesXml || consumesMultipart) {
-                for (Type parameter : methodInfo.parameterTypes()) {
-                    ClassInfo effectiveParameter = getEffectiveClassInfo(parameter, indexView);
+                for (MethodParameterInfo parameter : methodInfo.parameters()) {
+                    if (!isTypeCompatibleWithJaxb(parameter.type())) {
```

Comment:
Is this perhaps too general? I mean, by looking at the code it seems to not be limited to body parameters, but instead apply to all

Reply:
I know what you mean, and that's why I scratched the idea of doing it at first. 
Do you know if there is a utility or similar to know whether a parameter is an actual body parameter? One possible solution is to check the parameters that have no annotations. Yet it's not accurate enough.

Reply:
See https://github.com/quarkusio/quarkus/blob/158d4c6e39731b81b66c9553fe4b967c6bfadaf3/extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/QuarkusServerEndpointIndexer.java#L178

Reply:
PR updated using this.